src/App.js

```diff
@@ -1,25 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+import Card from './components/Card'
+import data from "./data"
+const App = () => {
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='bg-blue-50 flex p-12 gap-12 h-screen' >
+      {data.map((info, index) => {
+        return (<Card key={index} setup={info.setup} punchline={info.punchline} />)
+
```

Comment:
I think it's a good idea to use ID instead of the index in case you have IDs. because sometimes indexes can lead to bad performance in some cases! but in case you don't have IDs you can use them.
```suggestion
      {data.map(info => {
        return (<Card key={info.id} setup={info.setup} punchline={info.punchline} />)

```